praw/objects.py

```diff
@@ -640,15 +669,113 @@ def _update_submission(self, submission):
                 reply._update_submission(submission)  # pylint: disable=W0212
 
     @property
+    def default_contextlink(self):
```

Comment:
I don't see any need for `default_context_url` (assuming we change it to that), if there exists `context_url` that provides the default value.